src/py/flwr/cli/new/new.py

```diff
@@ -124,27 +125,37 @@ def new(
 
     # Set project directory path
     cwd = os.getcwd()
-    pnl = project_name.lower()
-    project_dir = os.path.join(cwd, pnl)
+    module_name = re.sub(r"[-_.]+", "-", project_name).lower()
```

Comment:
Should we remove `.` here?
```suggestion
    distribution_name = re.sub(r"[-_]+", "-", project_name).lower()
```

Reply:
Or should it be:
```suggestion
    module_name = re.sub(r"[-_.]+", "_", project_name).lower()
```

Reply:
Or should it be:
```suggestion
    package_name = re.sub(r"[-_.]+", "-", project_name).lower()
```